WordPress/src/main/res/values/strings.xml

```diff
@@ -1002,6 +1003,7 @@
     <!-- Stats accessibility strings -->
     <string name="stats_loading_card">Loading selected card data</string>
     <string name="stats_overview_content_description">%1$s %2$s for period: %3$s, change from previous period - %4$s</string>
+    <string name="stats_total_followers_content_description">%1s, %2s of total followers</string>
```

Comment:
@irfano I think there is a typo here, these `1` and `2` were probably meant to be positional parameters, aka `%1$s` and `%2$s` instead?

I spotted this while doing the release for 20.2 and seeing some warnings emitted by the compiler saying "Multiple substitutions specified in non-positional format of string resource string/stats_total_followers_content_description."

Reply:
Thanks for catching this @AliSoftware! 
This is content description text. I tested it with Talkback, and it works fine, but you're right. It should've been `%1$s`. I'll fix it in a new PR for 20.3.